spec/truffle/launcher_spec.rb

```diff
@@ -85,31 +85,31 @@ def should_print_full_java_command(cmd)
   end
 
   it "takes options from system properties set in JAVA_OPTS" do
-    out = `JAVA_OPTS=-Dpolyglot.ruby.verbosity=2 #{RbConfig.ruby} -e 'puts $VERBOSE'`
+    out = `JAVA_OPTS=-Dpolyglot.ruby.verbosity=true #{RbConfig.ruby} -e 'puts $VERBOSE'`
```

Comment:
To make the option more precise so only the enum values are allowed `nil,false,true` as in other enum options.